### PR TITLE
New sandbox tab in documentation

### PR DESCRIPTION
**Pull request by amirahaziz**
Added a new tab in our documentation that discusses our processes for Federalist sandboxes re: issue close #2025

### CODE REVIEW

**Comment by amirbey** [comment]
@amirahaziz - how will users access this page? do we need to add a link in the navigation or from within another page?

**Comment by amirahaziz** [comment]
@amirbey Hey Amir - we are intended on adding a new tab to the "Documentation" page. That is how users will access this. Will be its own page altogether. Did I make a wrong PR?

File: pages/documentation/federalist-sandbox.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+sidenav: documentation
+redirect_from: 
+  - /using-federalist/federalist-sandbox/
+---
+
+# {{ page.title }}
+
+## Getting started with Federalist sandbox environment:
+
+Having the ability to practice with the actual tool is important, and so with Federalist we give you the power to do just that, prior to any contracts or agreements. If you’re interested in seeing how Federalist works for yourself, we’re more than happy to set you up with a sandbox environment in Github. We don’t require a contract or agreement. This will allow you to practice using the tool itself, and to get comfortable with the platform before any of the real implementations or deployments take place. We understand federal contracting can be challenging, so read how AFWERX was able to build their entire website through the sandbox environment, while they waited for their legal departments to complete the contract. Having the sandbox environment saved them time, so that when the contract was signed, in less than 30 minutes their completed website was published live. This is just another added benefit of working with the Federalist program.
```
[the remark this answers is not in the document]
Comment (amirahaziz) [reply pages/documentation/federalist-sandbox.md]: Yes, intending to link it...but we don't have this up live yet. One of the things I am working on. Should we say "link coming soon?". Or can I link to a PDF and have it opened to a new page?

File: pages/documentation/federalist-sandbox.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+## When you’re ready to launch:
+
+If after using the sandbox proves Federalist is the solution with you, our team will work with you to begin the necessary contracting and IAA process. In the meantime, you’re more than happy to continue using the sandbox environment in staging. Once the IAA is signed and we have the necessary agreements in place, this is your greenlight to publish your sites live. Per compliance, we cannot allow websites to be pushed to a productive state until the necessary documents are signed and ready to go. However, we do encourage you to create and build whatever you’d like in the sandbox - it’s yours to explore!
```
[the remark this answers is not in the document]
Comment (amirahaziz) [reply pages/documentation/federalist-sandbox.md]: I think "this is your greenlight" is a good idea to add in here. I think the point of this section is that we want users to actually pay for the Federalist service, and not continue using the sandboxes for free

File: pages/documentation/federalist-sandbox.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+## When you’re ready to launch:
+
+If after using the sandbox proves Federalist is the solution with you, our team will work with you to begin the necessary contracting and IAA process. In the meantime, you’re more than happy to continue using the sandbox environment in staging. Once the IAA is signed and we have the necessary agreements in place, this is your greenlight to publish your sites live. Per compliance, we cannot allow websites to be pushed to a productive state until the necessary documents are signed and ready to go. However, we do encourage you to create and build whatever you’d like in the sandbox - it’s yours to explore!  
+
```
[the remark this answers is not in the document]
Comment (amirahaziz) [reply pages/documentation/federalist-sandbox.md]: You may need to explain this one to me...what are trailing lines?

File: pages/documentation/federalist-sandbox.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (amirbey) [reply pages/documentation/federalist-sandbox.md]: PDF for now could work? and then update it after? how far out is the story? can we make updating the link a checklist item in the story issue?

File: pages/documentation/federalist-sandbox.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (amirbey) [reply pages/documentation/federalist-sandbox.md]: i agree ... however, my concern is that once its signed it appears there are no more steps to publish site live ...  i could be wrong and making that up. if you want to stick with it as is, and you disagree, i am fine to move forward as is

File: pages/documentation/federalist-sandbox.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (amirahaziz) [reply pages/documentation/federalist-sandbox.md]: Yes, PDF will work! And then I will have a reminder to update

**Pull request merged**